ext/mri/2.3.0/id_table.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RUBY_ID_TABLE_H
+#define RUBY_ID_TABLE_H 1
+#include "ruby/ruby.h"
+
+struct rb_id_table;
```

Comment:
Yeah, I'm sorry that I won't be much help here, since I don't write C much either.